src/docs/perf/app-size.md

```diff
@@ -60,6 +73,12 @@ run the following:
 flutter build ios && tar -zcf build/app.ipa build/ios/iphoneos/Runner.app && ls -lh build/app.ipa
 ```
 
+The resulting IPA file for this example is 8.3 MB.
```

Comment:
Shall we be more specific?

The resulting IPA file for our `example/hello_world` app at the moment is 8.3 MB.